b2sdk/_v3/__init__.py

```diff
@@ -187,7 +187,9 @@
 from b2sdk.sync.encryption_provider import AbstractSyncEncryptionSettingsProvider
 from b2sdk.sync.encryption_provider import BasicSyncEncryptionSettingsProvider
 from b2sdk.sync.encryption_provider import ServerDefaultSyncEncryptionSettingsProvider
-from b2sdk.sync.encryption_provider import SERVER_DEFAULT_SYNC_ENCRYPTION_SETTINGS_PROVIDER
+from b2sdk.sync.encryption_provider import (
+    SERVER_DEFAULT_SYNC_ENCRYPTION_SETTINGS_PROVIDER,
+)
```

Comment:
This is stupid

Reply:
I think it's just because of width limit being too low? @vbaltrusaitis-reef

Reply:
Yes, black woudn't have split this had I formatted with `line-length=100` the first time. Now that there's a comma though, black will keep this the way it is regardless of `line-length` parameter.